README.md

```diff
@@ -74,15 +73,40 @@ git add -A && git commit
 Most important changes will be listed here, all other changes since `19.4.0` can be found in
 [release](https://github.com/OpenMage/magento-lts/releases) notes.
 
-### Performance
-<small>ToDo: Please add performance related changes as run-time cache, ...</small>
+### Between Magento 1.9.4.5 and OpenMage 19.x
+
+Bug fixes and PHP 7.x and 8.0 compatibility.
+
+### Between OpenMage 19.x and 20.x
+
+Do not use 20.x.x if you need IE support.
+
+- removed IE conditional comments, IE styles, IE scripts and IE eot files #1073
+- removed frontend default themes (default, modern, iphone, german, french, blank, blue) #1600
+- fixed incorrect datetime in customer block (`$useTimezone` parameter) #1525
+- add redis as a valid option for `global/session_save` #1513
+- possibility to disable global search in backend #1532
+
+For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/v19.4.15...v20.0.13).
+
+### Between OpenMage 20.x and 21.x
+
+Do not use 21.x.x if you don't know what you're doing, or if you don't understand consequences of the following changes.
+
+- removed mysql4 classes #1470
+- removed admin routing compatibility mode for extensions #1551
+
+For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/v20.0.13...v21.0.0).
```

Comment:
```suggestion
For full list of changes, you can [compare tags](https://github.com/OpenMage/magento-lts/compare/20.0...21.0).
```
Also, why are we documenting a version that doesn't exist yet?

Reply:
Because I hope it will be released.
In UTC time there are still some hours before 2022!

Reply:
Well yeah, I would prefer if it aligns with year i.e 22 instead of 21